tests/main/snapctl-services/task.yaml

```diff
@@ -9,7 +9,9 @@ restore: |
     rm -f $SERVICEOPTIONFILE
 
 execute: |
-    wait_for_service() {
+    # there is wait_for_service defined in systemd.sh, use a different name to
+    # avoid redefining or being obscured by the other one
+    _wait_for_service() {
```

Comment:
I wonder if this one should use the real wait_for_service from systemd.sh - but maybe for a followup PR.